tests/integration-tests/data-source-revert/package.json

```diff
@@ -7,7 +7,7 @@
     "deploy:test-grafted": "graph deploy test/data-source-revert-grafted grafted.yaml --version-label v0.0.1 --ipfs $IPFS_URI --node $GRAPH_NODE_ADMIN_URI"
   },
   "devDependencies": {
-    "@graphprotocol/graph-cli": "https://github.com/graphprotocol/graph-cli#main",
-    "@graphprotocol/graph-ts": "https://github.com/graphprotocol/graph-ts#main"
+    "@graphprotocol/graph-cli": "latest",
+    "@graphprotocol/graph-ts": "latest"
```

Comment:
I think this is better and safer cause essentially we want to try new features

Reply:
Agree!

Reply:
I like the idea, but we must find another way to periodically upgrade our dependencies without `latest`. `yarn.lock` exists specifically to make builds as deterministic as possible, and an upstream change should never break our builds (case in point, CI fails now). I'm working on a PoC with a Dependabot-like workflow, so I'll post more information here. Until then, all of these version changes have to be reverted so the CI succeeds.

Reply:
Alright, I think I got it. `latest` can stay. We just need to 1. modify `dependabot.yml` and 2. rebuild `yarn.lock`.
@saihaj you can cherrypick from my branch `filippo/yarn-lock-dependabot` if necessary (`https://github.com/graphprotocol/graph-node/commit/592aeae2f6b204b991fc2334161045ad761b583c` and `https://github.com/graphprotocol/graph-node/commit/b1d3633c9c87aedf100f060a99b1e8da7daf17b9`).